pkg/gui/command_log_panel.go

```diff
@@ -44,14 +43,14 @@ func (gui *Gui) printCommandLogHeader() {
 		gui.Tr.CommandLogHeader,
 		gui.getKeyDisplay(gui.Config.GetUserConfig().Keybinding.Universal.ExtrasMenu),
 	)
-	fmt.Fprintln(gui.Views.Extras, utils.ColoredString(introStr, color.FgCyan))
+	fmt.Fprintln(gui.Views.Extras, color.FgCyan.Sprint(introStr))
```

Comment:
I believe we should we be using `theme.FgCyan` here. In fact it would be good to have all our references to the color package live in one place so that it'll be easy to change the library again in the future if need be

Reply:
The github.com/gookit/color should now be gone in all package